codersdk/presets.go

```diff
@@ -15,6 +15,7 @@ type Preset struct {
 	Name       string
 	Parameters []PresetParameter
 	Default    bool
+	Prebuilds  *int
```

Comment:
Consider clarifying the naming to communicate whether this is the desired number of prebuilds or the eligible number of prebuilds. I can see from elsewhere that its the desired number, but we've spoken in the past about showing how many prebuilds are eligible in the frontend before. Would be good to leave room for both.

Reply:
Changed to `DesiredPrebuildInstances`. Addressed in: https://github.com/coder/coder/pull/18910/commits/641dc96e63dccee8c2984b3d946231269cc50584